packages/esm-patient-allergies-app/src/allergies/allergies-tile.component.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useAllergies } from './allergy-intolerance.resource';
+import { useTranslation } from 'react-i18next';
+import styles from './allergies-tile.component.scss';
+
+interface AllergyTileInterface {
+  patientUuid: string;
+}
+
+const AllergyTile: React.FC<AllergyTileInterface> = ({ patientUuid }) => {
+  const { data: allergies } = useAllergies(patientUuid);
+  const { t } = useTranslation();
+
+  return (
+    <div className={styles.gridColumn}>
+      <p>{t('allergies', 'Allergies')}</p>
+      <span>
+        {allergies
+          ?.map((allergy) => allergy?.reactionManifestations)
```

Comment:
This is not the data that should be shown for allergies. The data that should be shown is the thing that the patient is allergic to.